Services/JenkinsDocker/JenkinsSF/ApplicationManifest.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
yeah. I saw your follow up on the thread. that makes sense. But for now, let's have things totally updated here. So that if we need to move to a new repo may be this TP or next, we can do that easily.

Also, just uploading the jenkins.sh and dockerfile would not be an ideal way to open-source that. Generally the way it is done is uploading the full-folder in the supposedly new repo (though there may not be changes in other files).